packages/manager/apps/dedicated/client/app/dedicated/server/display-name/display-name.controller.js

```diff
@@ -61,8 +56,16 @@ angular.module('App').controller(
             ].join('. '),
             'server_dashboard_alert',
           );
-          this.$state.go('^');
         })
+        .finally(() =>
+          this.$state.go(
+            '^',
+            {},
+            {
+              reload,
+            },
+          ),
+        )
```

Comment:
@anooparveti, for migration raisons, we have to not implement $state in controllers.
If duable you have to write it into a routing file.

Also the reload have to be implement into then promise.
```suggestion
        .then(() =>
          this.refreshPage(),
        ).catch(() => {  })
```

Reply:
Hello @mohammed-zahaf, I agree with your suggestion of moving `$state` to routing file but unfortunately this particular state is legacy and is implemented without `component` architecture. So it cannot be implemented as of now.